src/quacc/calculators/espresso/espresso.py

```diff
@@ -424,9 +424,13 @@ def __init__(
             .get("pseudo_dir", str(SETTINGS.ESPRESSO_PSEUDO))
         )
 
+        prefix = (
+            os.environ.get("PARSL_MPI_PREFIX", "") or SETTINGS.ESPRESSO_PARALLEL_CMD[0]
```

Comment:
Small nit: You can just do `os.environ.get("PARSL_MPI_PREFIX")` without assigning a default. The default is `None`.

Reply:
Yes, I don't know what happened yesterday evening but I just got confused...